guides/common/modules/proc_installing-the-load-balancer.adoc

```diff
@@ -1,21 +1,21 @@
 [id="Installing_the_Load_Balancer_{context}"]
 = Installing the load balancer
 
-The following example provides general guidance for configuring an HAProxy load balancer using {EL} 9 or {EL} 8.
+The following example provides general guidance for configuring an HAProxy load balancer using {EL} 9.
 However, you can install any suitable load balancing software solution that supports TCP forwarding.
 
 .Procedure
 . Install HAProxy:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {client-package-install-el8} haproxy
+# {client-package-install-el9} haproxy
 ----
 . Install the following package that includes the `semanage` tool:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {client-package-install-el8} policycoreutils-python-utils
+# {client-package-install-el9} policycoreutils-python-utils
```

Comment:
I'm not sure about these. I thought that a load balancer needs to be installed on a smart proxy, but these procedures use client-package-install-el8. Should I leave this whole file as it was?

Reply:
I think `client-package-install-el8` was wrong in the first place. Since we dropped EL7 we should really simply it back to just `server-package-install`, which is defined different for EL vs deb but otherwise always the same.

Reply:
Are you sure? My understanding is that you install HAProxy on a fresh EL 8 host which does not yet use `satellite-maintain` to install packages? See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L102

Reply:
I think you're right, but that's why the client vs server is so confusing. HAProxy will be a server and probably not a client for Foreman (though it might)

Reply:
It looks like we should track this separately so I'd appreciate your review here: https://github.com/theforeman/foreman-documentation/pull/3605.

I'll drop the change from this PR.